apps/s_time.c

```diff
@@ -240,16 +224,10 @@ int s_time_main(int argc, char **argv)
                                    www_path);
             if (buf_len <= 0 || SSL_write(scon, buf, buf_len) <= 0)
                 goto end;
-            while ((i = SSL_read(scon, buf, sizeof(buf))) > 0 ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_READ ||
-                        SSL_get_error(scon, i) == SSL_ERROR_WANT_WRITE)
```

Comment:
> Can you please explain why the SSL not simply reads the data?

Because there is no data to read. The arrival of the NewSessionTicket message means that a record gets processed by libssl, but it does not yield any application data.

> I mean this is obviously an API change, isn't it?

Nope. This has always been the case - its just been made more obvious by TLSv1.3. For example in TLSv1.2 this could happen if a reneg occurs.